packages/preview/sourcerer/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "sourcerer"
+version = "0.1.0"
+authors = ["Yoav Grimland <miestrode@proton.me>"]
+license = "MIT"
+description = "Show code in a flexible, customizable way."
+repository = "https://github.com/miestrode/sourcerer"
+
+keywords = ["code", "sourcecode", "listing", "listings"]
+
+entrypoint = "src/lib.typ"
+compiler = "0.9.0"
+
+exclude = ["assets/*", "DOCS.md", "README.md", "LICENSE"]
```

Comment:
the LICENSE should definitely not be excluded as it should be distributed alongside the sources. I also would not exclude the README or DOCS. `exclude` is really just meant for heavy files.

Reply:
Ah, I thought it would be fine as I also specify things in `typst.toml`. I'll change it.